src/lib/element.html

```diff
@@ -0,0 +1,56 @@
+<!--
+@license
+Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<link rel="import" href="base.html">
+<script>
+(function() {
+
+  'use strict';
+
+  /**
+   * A base class for class-based Polymer elements.
+   *
+   *     class MyElement extends Polymer.Element {
+   *       static get is() { return ‘my-element’; }
+   *       static get config() { return {
+   *         properties: {
+   *           // the properties you know and love from Polymer
+   *         },
+   *         observers: [
+   *           // the observers you know and love from Polymer
+   *         ]
+   *       }}
+   *       get listeners() { return {
+   *         // the listeners you know and love from Polymer
+   *       }}
+   *     }
+   *
+   * Element classes should extend directly from Polymer.Element. Property,
+   * observer, and listener declarations on superclasses will not be processed
+   * by Polymer.
+   */
+  function PolymerElement() {}
+  PolymerElement.prototype = {
+    get is() { return this.constructor.is; },
+    get properties() {
+      var config = this.constructor.config;
+      return config && config.properties;
+    },
+    get observers() {
+      var config = this.constructor.config;
+      return config && config.observers;
+    },
+  };
+  PolymerElement.prototype.constructor = PolymerElement;
+  Object.setPrototypeOf(PolymerElement.prototype, Polymer.Base);
+  Object.setPrototypeOf(PolymerElement, HTMLElement);
```

Comment:
Eh, let's go ahead and use `Polymer.Base.chainObject` (same as https://github.com/Polymer/polymer/pull/4107/files#diff-7838a926e8a75d4e50eb319e1284438dR61), since there's no good reason not to...